data/abilities.js

```diff
@@ -1408,6 +1481,7 @@ var abilityList = [
 	{
 		id: 592,

```

Comment:
@Zeffuro I'm a little confused about this Buff, are you sure it's a MNK's Buff?

Reply:
Not sure, I don't have all jobs at max. Might need to remove this but I'll do some research to decide on that.